Changelog.md

```diff
@@ -11,6 +11,7 @@ Compiler Features:
 Bugfixes:
  * isoltest: Added new keyword `wei` to express function value in semantic tests
  * Standard-JSON-Interface: Fix a bug related to empty filenames and imports.
+ * ipfs-hashing: Added support for large files that we need to split in multiple chunks.
```

Comment:
I think this should rather do to the `Compiler Features` section

Reply:
Moved